pkg/archive/archive_test.go

```diff
@@ -74,15 +74,6 @@ func testArchive(t *testing.T, when spec.G, it spec.S) {
 				verify.NextDirectory("/nested/dir/dir-in-archive", int64(os.ModePerm))
 			})
 		})
-
```

Comment:
Note: this test does not make sense anymore since the employed docker function does check for double-close itself.